pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -587,6 +597,21 @@ func (cr *PerconaXtraDBCluster) CheckNSetDefaults(serverVersion *version.ServerV
 	}
 
 	if c.Backup != nil {
+		if c.Backup.Image == "" {
+			return false, fmt.Errorf("backup.Image can't be empty")
+		}
```

Comment:
I believe all checks are moved to `cr.Validate` function. we should follow this everywhere

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/673/commits/29cb7351354859be1f85f2f66e688acc406af6f2